examples/data-sources/kentik-synthetics_agents/run.sh

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env bash
-# Run the example using live Kentik API server.
+# The script applies the example Terraform configuration.
+# The provider uses stub Kentik API server by default.
+# Production Kentik API server can be used by passing "production" positional argument to the script.
 
 SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
 REPO_DIR=$(cd -- "$SCRIPT_DIR" && cd ../../../ && pwd)
 
+PRODUCTION_FLAG="production"
+TEST_API_SERVER_ENDPOINT=${TEST_API_SERVER_ENDPOINT:-"localhost:9955"}
+
 function run() {
+    if [[ $1 == "$PRODUCTION_FLAG" ]]; then
+        stage "Running the example using production Kentik API server"
+        warn "Warning: Kentik production resources might be modified and paid actions might be invoked"
```

Comment:
There actually no "paid actions in Kentik". There are though resources which have consumption limit based on purchased product/package". Synthetic test credits is one of them. So, running the test script against Kentik production cannot not result in immediate cost increase, rather it may prevent other tests from running due to shortage of credits.

Reply:
Removed "paid actions in Kentik" part